docs/plugins/dashboard/gatsby/on-create-pages.js

```diff
@@ -37,9 +38,19 @@ module.exports = async ({ graphql, actions, reporter, cache }) => {
       };
     });
 
+    pages.push({
```

Comment:
Best make it the first item.
```suggestion
    pages.unshift({
```